src/components/Layout/Layout.tsx

```diff
@@ -31,23 +31,23 @@ type LayoutProps = PageProps<unknown, PageContextType>;
 
 const Layout: React.FC<LayoutProps> = ({ children, pageContext }) => {
   const location = useLocation();
-  const { searchBar, leftSidebar, rightSidebar, template } = pageContext.layout ?? {};
+  const { leftSidebar, rightSidebar, template } = pageContext.layout ?? {};
   const isRedocPage =
     location.pathname === '/docs/api/control-api' ||
     location.pathname === '/docs/api/chat-rest' ||
     location.pathname === '/docs/api/liveobjects-rest';
 
   return (
     <GlobalLoading template={template}>
-      <Header searchBar={searchBar} />
-      <div className="flex pt-16 md:gap-12 lg:gap-16 xl:gap-20 justify-center ui-standard-container mx-auto">
-        {leftSidebar ? <LeftSidebar /> : null}
+      <Header />
+      <div className="flex pt-16 px-12 md:px-0 md:gap-12 lg:gap-16 xl:gap-20 justify-center">
+        {leftSidebar ? <LeftSidebar /> : <div />}
```

Comment:
[question] Empty `div`'s to hold the grid in shape?

Reply:
That's right